code/game/gamemodes/clock_cult/clock_effects/clock_sigils.dm

```diff
@@ -20,7 +20,7 @@
 	return ..()
 
 /obj/effect/clockwork/sigil/attack_hand(mob/user)
-	if(iscarbon(user) && !user.stat && (!is_servant_of_ratvar(user) || (is_servant_of_ratvar(user) && user.a_intent == INTENT_HARM)))
```

Comment:
I'm not trying to cause any conflict, just to give my thoughts. Cultists have a way to remove runes and their own structures; should servants not?